BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-98: Form - admins

- Add a redux thunk to GET the entity by its ID
- Implement react-hook-form
- Implement resolver
- Implement Joi validations (see description)
- Implement error messages for validations

QA: 
To test this ticket we should open the link of the app deployed in Vercel. Then, we should verify the correct functionality of:

- In the admins' form, all fields must show error messages when the inputs don't pass the vaidations. In case the form is called by an update button, the fields must contain the data charged.
- Check if all validations work correctly and show the respective errors.
- The submit button must save the information.
-Reset button should reset the inputs data

The objective of this test is to verify the correct functionality of the forms after the implementation of react-hook-form and joi
